anymaniax/orval
# feat: generate mocks for all responses

## Status



**READY**

## Description

Generate mocks for all responses defined in an openapi spec.

Fixes: #1280.

## Todos


## Steps to Test or Reproduce

1. cd to `samples/react-query/status-codes`
2. Run `yarn generate-api`.
3. Open `samples/react-query/status-codes/src/api/endpoints/petstoreFromFileSpecWithTransformer.msw.ts`
4. See that mocks and http handlers are generated for each response.
5. At the bottom see that only `getListPetsMockHandler()` is included in the array of handlers (this is so that when passing the handlers to the worker there is only one http response for each path).
